slf4j/src/main/scala/org/typelevel/log4cats/slf4j/Slf4jLogger.scala

```diff
@@ -33,10 +34,10 @@ object Slf4jLogger extends Slf4jLoggerCompat {
     getLoggerFromSlf4j[F](org.slf4j.LoggerFactory.getLogger(clazz))
 
   def getLoggerFromSlf4j[F[_]: Sync](logger: JLogger): SelfAwareStructuredLogger[F] =
-    new Slf4jLoggerInternal.Slf4jLogger(logger, Sync.Type.Delay)
+    new Slf4jLoggerInternal.Slf4jLogger(logger, Sync.Type.Delay, Applicative[F].pure(Map.empty))
```

Comment:
This (and line 40) wire nothing in yet.  We'd need a parallel set of Ask or Local based constructors here.